harvester/records/record.py

```diff
@@ -343,16 +348,39 @@ def normalize(self) -> MITAardvark:
                     logger.exception(message)
                     raise FieldMethodError(exc, message) from exc
 
-        # dedupe all list fields
-        for field_name, field_values in all_field_values.items():
-            if isinstance(field_values, list):
-                deduped_field_values = [
-                    value for value in field_values if value is not None
-                ]
-                all_field_values[field_name] = dedupe_list_of_values(deduped_field_values)
+        # post normalization quality improvements
+        for field_name, original_value in all_field_values.items():
+            clean_value = self._remove_none_and_blank_strings(original_value)
+            clean_value = self._dedupe_list_fields(clean_value)
+            all_field_values[field_name] = clean_value
 
         # initialize a new MITAardvark instance and return
-        return MITAardvark(**all_field_values)
+        return MITAardvark(**all_field_values)  # type: ignore[arg-type]
+
+    @staticmethod
+    def _remove_none_and_blank_strings(
+        original_value: MITAardvarkFieldValue,
+    ) -> MITAardvarkFieldValue:
+        """Remove None values and empty strings from MITAardvark field value."""
+        if isinstance(original_value, str):
+            return None if original_value.strip() == "" else original_value
```

Comment:
[Non-blocking] In reviewing the data, have you encountered any cases where value = "    banana"? It might be nice if we could also return values where leading and trailing zeros are stripped but value `!= ""`.

Reply:
@jonavellecuerdo - Stripping leading/trailing whitespace is a good idea.  This happens to some degree in the metadata format normalizations, but I'm fully on board with focusing more heavily on the "post" cleanup both here in GeoHarvester and Transmogrifier.  Might save that for a later PR.